styles.css

```diff
@@ -1 +1,601 @@
- /*  Hier wordt de CSS Concept opdracht les 7 gemaakt.  */
\ No newline at end of file
+/*------------------------------------------ CSS Concept opdracht les 7 ---------------------------------------------*/
+
+@import url('https://fonts.googleapis.com/css2?family=Merriweather&family=Roboto:wght@300;900&display=swap');
+
+/*   CSS reset  */
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+/*-------------------------------------------------------------------------------------------------------------------*/
+body {
+    width: 100vw;
+}
+
+/*  variable primary-color  */
+:root {
+    --primary-color-purple: #5722CD;
+    --primary-color-bleu: #4845E4;
+    --primary-color-pink: #ED4385;
+    --primary-color-black: #0C0C0C; /* voor standaard tekst */
+    --primary-color-purple-light: #F2F0FF; /* voor grote zwevende woorden "hello"en "services" */
+    --primary-color-grey-light: #F3F6F9; /* voor grote zwevende woorden "hello"en "services" */
+    --primary-color-grey: #7A7A7A; /* voor de service-regel tekst" */
+}
+
+/*------------------- Reusable class ----------------------*/
+
+.column {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.row {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+/*--------------------- Begin header CSS -----------------------*/
+
+/*  Clusteren  */
+header, h1, h2 {
+    font-family: 'Merriweather', serif;
+    color: white;
+}
+
+header h1 {
+    text-align: center;
+    margin: 0.2em;
+}
+
+header p {
+    font-family: 'Roboto', sans-serif;
+    color: white;
+    font-size: 26px;
+}
+
+.outer-container {
+    background: var(--primary-color-purple);
+    background: linear-gradient(327deg, var(--primary-color-purple) 0%, var(--primary-color-bleu) 100%);
+    height: 600px;
+}
+
+.navbar > ul {
+    gap: 60px;
+    margin: 50px 0 100px 0;
+}
+
+.navbar li {
+    list-style: none;
+}
+
+.navbar a {
+    text-decoration: none;
+    color: white;
+}
+
+.hero-section {
+    gap: 40px;
+}
+
+.cta-btn {
+    text-transform: uppercase;
+    font-size: 14px;
+    border-radius: 30px;
+    background-color: var(--primary-color-pink);
+    border: none;
+    padding: 20px;
+    text-align: center;
+    display: inline-block;
+    margin: 20px 2px 40px;
+    cursor: pointer;
+}
+
+.cta-btn a {
+    color: white;
+    text-decoration: none;
+}
+
+.squiggle-line-stripe {
+    margin-top: 30px;
+}
+
+.triangle-bottom-left {
+    width: 0;
+    height: 0;
+    border-style: solid;
+    border-width: 98px 0 0 100vw;
+    border-color: transparent transparent transparent #fff;
+    margin-top: -98px;
+}
+
+/*-------------End header CSS------------------*/
+
+/*-------------Begin main CSS------------------*/
+.triangle-top-right {
```

Comment:
Interessante oplossing! Leuk en creatief :) _alleen…_ als ik de website kleiner maak (bijvoorbeeld op de mobile version) wordt de hoek groter. Kleine tip: kijk eens in CSS naar transform property: https://www.w3schools.com/cssref/css3_pr_transform.asp